### ISSUE TITLE
Create a standalone Docker container with fully functional app and some synthetic data

### ISSUE THREAD

**Issue by mojca** [macports/macports-webapp#307]
I'm putting this idea up for discussion / brainstorming / thoughts.

If someone with little experience or little time would like to make some simple contributions to the application, it's currently somewhat non-trivial to set up and populate the database to get a demo app up and running quickly.

It seems to me that it would be extremely useful if users on any operating system could be able to quickly run a few commands (not having to bother with PostgreSQL and SOLR setup and administration on their main OS) and have the development environment ready with little effort, getting a fully functional app out of the box.

We probably overlooked the volumes functionality of Docker when we were first discussing how to set up the app two years ago:
https://docs.docker.com/storage/volumes/

(This is something that might have been useful also for the main app, but we would certainly need it for this.)

We need to do two separate things to achieve this:
* prepare `Dockerfile` and documentation (most important)
* write a python script to populate the statistics data with some random data (less important and can be done any time later)

**Comment by amake** [comment]
I set the webapp up for the first time today and I really would have appreciated a docker-compose.yml that did a one-command setup. Both Solr and Postgres have official Docker images, so it's "just" a matter of orchestrating things properly.

**Comment by arjunsalyan** [comment]
Fair point, a docker-compose can simplify local setup and development to a huge extent.
